### ISSUE TITLE
Adicionar .por ao invés de .pb no idioma PT-BR

### ISSUE THREAD

**Issue by mafhper** [subtitle-master/subtitlemaster#2]
Olá,

seria legal se ao renomear o arquivo *.srt, foi adicionado sufixo *.por ao invés de *.pb para o Português do Brasil.

**Comment by ricardofunke** [comment]
O certo seria pt_BR, mas eu preferia sem sufixo para não precisar renomear já que os players pegam a legenda automaticamente se tiverem exatamente o mesmo nome do filme

**Comment by mafhper** [comment]
Sim, pode ser. Talvez uma toggle de opção para isso. Selecionando com ou sem, e se ativo qual origem. 

Na versão anterior do Subtitle Master a legenda saia sem o sufixo. Dali eu jogava na minha pasta de script e já juntava tudo com o subler.
